internal/pkg/metadb/blobref/blobref.go

```diff
@@ -71,7 +71,16 @@ func (b *BlobRef) Save() ([]datastore.Property, error) {
 // Load implements the Datastore PropertyLoadSaver interface and converts Datstore
 // properties to the Properties field.
 func (b *BlobRef) Load(ps []datastore.Property) error {
-	return datastore.LoadStruct(b, ps)
+        // ps_modified has been added to enable backward compatibility by creating 
+        // a new datastore array that replaces legacy "NumberOfChunks" with "ChunkCount". 
+        var ps_modified []datastore.Property
```

Comment:
You can directly change `ps` I think. If you need to use a new slice, please use mixedCaps.

https://go.dev/doc/effective_go#names

Reply:
I checked in the history, and "NumberOfChunks" has never been a field of BlobRef. When introduced, it was called ChunkCount. You can remove this change.